classes/03class/exercises/c03-aws02/mrcsmonteiro/README.md

```diff
@@ -0,0 +1,203 @@
+            ],
+            "SecurityGroups": [
+                "sg-010e9583a629eae27"
+            ],
+            "IpAddressType": "ipv4"
+        }
+    ]
+}
+
+$ aws elbv2 create-target-group \
+    --name devops-academy-tg \
+    --protocol HTTP \
+    --port 80 \
+    --target-type instance \
+    --vpc-id vpc-0721e28436cea8410
+
+{
+    "TargetGroups": [
+        {
+            "TargetGroupArn": "arn:aws:elasticloadbalancing:ap-southeast-2:149613515908:targetgroup/devops-academy-tg/16871e3110304536",
+            "TargetGroupName": "devops-academy-tg",
+            "Protocol": "HTTP",
+            "Port": 80,
+            "VpcId": "vpc-0721e28436cea8410",
+            "HealthCheckProtocol": "HTTP",
+            "HealthCheckPort": "traffic-port",
+            "HealthCheckEnabled": true,
+            "HealthCheckIntervalSeconds": 30,
+            "HealthCheckTimeoutSeconds": 5,
+            "HealthyThresholdCount": 5,
+            "UnhealthyThresholdCount": 2,
+            "HealthCheckPath": "/",
+            "Matcher": {
+                "HttpCode": "200"
+            },
+            "TargetType": "instance"
+        }
+    ]
+}
+
+$ aws elbv2 add-tags \
+    --resource-arns arn:aws:elasticloadbalancing:ap-southeast-2:149613515908:targetgroup/devops-academy-tg/16871e3110304536 \
+    --tags "Key=Name,Value=DevOps Academy TG"
+
+$ aws elbv2 create-listener \
+    --load-balancer-arn arn:aws:elasticloadbalancing:ap-southeast-2:149613515908:loadbalancer/app/devops-academy-alb/d706929f7cd85a67 \
+    --protocol HTTP \
+    --port 80 \
+    --default-actions Type=forward,TargetGroupArn=arn:aws:elasticloadbalancing:ap-southeast-2:149613515908:targetgroup/devops-academy-tg/16871e3110304536
+
+{
+    "Listeners": [
+        {
+            "ListenerArn": "arn:aws:elasticloadbalancing:ap-southeast-2:149613515908:listener/app/devops-academy-alb/d706929f7cd85a67/c600e5ca31b516e4",
+            "LoadBalancerArn": "arn:aws:elasticloadbalancing:ap-southeast-2:149613515908:loadbalancer/app/devops-academy-alb/d706929f7cd85a67",
+            "Port": 80,
+            "Protocol": "HTTP",
+            "DefaultActions": [
+                {
+                    "Type": "forward",
+                    "TargetGroupArn": "arn:aws:elasticloadbalancing:ap-southeast-2:149613515908:targetgroup/devops-academy-tg/16871e3110304536",
+                    "ForwardConfig": {
+                        "TargetGroups": [
+                            {
+                                "TargetGroupArn": "arn:aws:elasticloadbalancing:ap-southeast-2:149613515908:targetgroup/devops-academy-tg/16871e3110304536",
+                                "Weight": 1
+                            }
+                        ],
+                        "TargetGroupStickinessConfig": {
+                            "Enabled": false
+                        }
+                    }
+                }
+            ]
+        }
+    ]
+}
+
+$ aws autoscaling attach-load-balancer-target-groups \
```

Comment:
Well done for using the ASG/Target group association. That's the way we do to auto-register new instances in the LB.